package.json

```diff
@@ -15,7 +15,7 @@
   },
   "homepage": "https://github.com/okta/oktasdk-sdk-dotnet#readme",
   "devDependencies": {
-    "@okta/openapi": "0.6.0",
+    "@okta/openapi": "^0.8.0",
```

Comment:
My preference is to explicitly pin this so it never changes without us knowing. Is there a reason you want `^`?

Reply:
nope, good catch